gherkin/java/src/main/java/io/cucumber/gherkin/Gherkin.java

```diff
@@ -50,9 +52,7 @@ private Gherkin(List<String> paths, List<Envelope> envelopes, boolean includeSou
     }
 
     public static Envelope makeSourceEnvelope(String data, String uri) {
```

Comment:
This method is only used in tests. Consider removing it to reduce the exposed API.